commander3/src/comm_tod_noise_mod.f90

```diff
@@ -820,6 +834,23 @@ subroutine sample_noise_psd(self, handle, scan, mask, s_tot, n_corr)
     ndet = self%ndet
     nomp = omp_get_max_threads()
 
+    if (trim(self%freq) == '030') then
```

Comment:
nooooooooooo

Reply:
I agree, this caused me pain. What's a good alternative?

Reply:
We'll add a prior argument to the routine, so that every caller will have to define its own priors explicitly, and not hardcode them here. The same applies to the gain jump list